compiler-rt/test/sanitizer_common/TestCases/Linux/dump_registers_aarch64.cpp

```diff
@@ -0,0 +1,23 @@
+// Check that sanitizer prints registers dump_registers on dump_registers=1
+// RUN: %clangxx  %s -o %t
+// RUN: %env_tool_opts=dump_registers=0 not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-NODUMP
+// RUN: not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-DUMP
```

Comment:
Please use `--strict-whitespace` for these tests

Reply:
Okay, thx. In this commit i'll add this cmdline option to arm and aarch64 tests and in the following commit for x86 and x86-64 tests